frontend/src/pages/functional_components/NewExperimentFC.tsx

```diff
@@ -118,8 +109,9 @@ export function NewExperimentFC(props: NewExperimentFCProps) {
         title: 'Experiment creation failed',
       });
     }
+    // Do not rerun this effect if updateDialog callback has changes to avoid re-rendering.
```

Comment:
Is this line of comment used to avoid rerun the useEffect()?

Reply:
This comment is an explanation for disabling eslint `react-hooks/exhaustive-deps`. Usually we try to fulfill the requirement of eslint by adding all required dependencies to the dependency list. But in this case, we are skipping some dependency, this line of comment is to explain what variable we have skipped and the reason.